client/service/src/builder.rs

```diff
@@ -534,6 +536,14 @@ where
 		metrics_service.run(client.clone(), transaction_pool.clone(), network.clone()),
 	);
 
+	if let Some(storage_monitor_service) = StorageMonitorService::new_for_config(&config)? {
+		task_manager.spawn_essential_handle().spawn(
```

Comment:
there is `statvfs` syscall called inside, so maybe this should be `spawn_blocking`?

Reply:
on my pc test running for few minutes: typically few µs, sometimes several dozes µs, rarely hundreds.

Histogram (in `nanoseconds`):
```# Number of samples = 147332815
# Min = 870
# Max = 124399
#
# Mean = 972.8840301798245
# Standard deviation = 243.63354324050133
# Variance = 59357.30339192123
#
# Each ∎ is a count of 2946629
#
   870 ..  13223 [ 147331481 ]: ∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎∎
 13223 ..  25576 [      1237 ]: 
 25576 ..  37929 [        48 ]: 
 37929 ..  50282 [        18 ]: 
 50282 ..  62635 [        15 ]: 
 62635 ..  74988 [         7 ]: 
 74988 ..  87341 [         3 ]: 
 87341 ..  99694 [         2 ]: 
 99694 .. 112047 [         1 ]: 
112047 .. 124400 [         3 ]: 
```

Still even it is low on average, sometimes it can block for a little bit longer.